### PR TITLE
ADCM-1844_3 add tests for сluster list page

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/page/cluster_list/page.py [hunk 1]
```diff
@@ -51,3 +58,53 @@ def get_cluster_info_from_row(self, row: int) -> dict:
...
+    def run_action_in_cluster_row(self, row: WebElement, action_name: str):
+        self.click_action_btn_in_row(row)
+        self.wait_element_visible(self.table.table.ActionPopup.block)
+        self.find_and_click(self.table.table.ActionPopup.button(action_name))
+        self.wait_element_visible(ActionDialog.body)
+        self.find_and_click(ActionDialog.run)
+
+    @contextmanager
+    def wait_cluster_state_change(self, row: WebElement):
+        state_before = self.find_child(row, self.table.table.ClusterRow.state).text
```
Comment (Sealwing) [inline tests/ui_tests/app/page/cluster_list/page.py:78]: `get_cluster_state_from_row` can be used here, am I right?
This way we have more predictable behavior

File: tests/ui_tests/app/page/cluster_list/page.py [hunk 2]
```diff
@@ -51,3 +58,53 @@ def get_cluster_info_from_row(self, row: int) -> dict:
...
+            assert state_after != state_before and state_after != self.table.loading_state_text
+
+        wait_until_step_succeeds(wait_state, period=1, timeout=self.default_loc_timeout)
+
+    @allure.step("Get cluster state")
+    def get_cluster_state_from_row(self, row: WebElement):
+        return self.find_child(row, self.table.table.ClusterRow.state).text
+
+    @allure.step("Get row by cluster name '{cluster_name}'")
+    def get_row_by_cluster_name(self, cluster_name: str) -> list:
```
Comment (Sealwing) [inline tests/ui_tests/app/page/cluster_list/page.py:92]: I think the return type is WebElement or something like it, not list?

File: tests/ui_tests/app/page/common/base_page.py [hunk 3]
```diff
@@ -228,6 +230,10 @@ def wait_config_loaded(self):
         self.find_element(CommonLocators.socket, timeout=30)
         self.find_element(CommonLocators.profile, timeout=30)
 
+    def hover_element(self, locator: Locator):
```
Comment (Sealwing) [inline tests/ui_tests/app/page/common/base_page.py:233]: Maybe add docstring here?
Hover sounds like putting the mouse over an element, but `move_to_element` associates with scrolling.
A bit confusing for a method in `base_page`

File: tests/ui_tests/app/page/common/table/page.py [hunk 4]
```diff
@@ -13,6 +13,8 @@
 class CommonTableObj(BasePageObject):
     """Class for manipulating with common tables elements."""
 
+    loading_state_text = 'autorenew'
```
Comment (Sealwing) [inline tests/ui_tests/app/page/common/table/page.py:16]: If it's constant, uppercase can clarify it

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: tests/ui_tests/app/page/common/table/page.py (hunk 4 above)
Comment (annademina) [reply tests/ui_tests/app/page/common/table/page.py]: changed

File: tests/ui_tests/app/page/cluster_list/page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/page/cluster_list/page.py]: thnx! changed

File: tests/ui_tests/app/page/cluster_list/page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/app/page/cluster_list/page.py]: thnx! changed

File: tests/ui_tests/app/page/common/base_page.py (hunk 3 above)
Comment (annademina) [reply tests/ui_tests/app/page/common/base_page.py:233]: added

**Review by Sealwing — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Pull request merged**